website/surveys.py

```diff
@@ -0,0 +1,51 @@
+from flask import request
+from .database import Database
+from .website_module import WebsiteModule, route
+
+
+class SurveysModule(WebsiteModule):
+    def __init__(self, db: Database):
+        super().__init__("surveys", __name__, url_prefix="/surveys")
+        self.db = db
+
+    @route("/submit-survey/<survey_id>", methods=['POST'])
+    def submit_survey(self, survey_id):
+        survey_done = True
+        survey = self.db.get_survey(survey_id)
+        responses = survey.get('responses')
+        if not responses:
+            responses = {}
+
+        for question, answer in request.form.items():
+            if not answer:
+                survey_done = False
+            responses[question] = answer
```

Comment:
This is also a good point! I didn't try switching languages between sessions. Maybe this will also prevent errors with respect to translators messing with the format.

Reply:
done!